addons/mail/static/src/components/activity/activity.xml

```diff
@@ -74,7 +74,7 @@
                                 <div t-if="activityView.activity.assignee" class="d-md-table-row mb-3">
                                     <div class="d-md-table-cell font-weight-bold text-md-right m-0 py-md-1 px-md-4">Assigned to</div>
                                     <div class="o_Activity_detailsAssignation d-md-table-cell py-md-1 pr-4">
-                                        <img class="o_Activity_detailsUserAvatar o_Activity_detailsAssignationUserAvatar" t-attf-src="/web/image/res.users/{{ activityView.activity.assignee.id }}/avatar_128" t-att-title="activityView.activity.assignee.nameOrDisplayName" t-att-alt="activityView.activity.assignee.nameOrDisplayName"/>
+                                        <img class="o_Activity_detailsUserAvatar o_Activity_detailsAssignationUserAvatar o_avatar_inline" t-attf-src="/web/image/res.users/{{ activityView.activity.assignee.id }}/avatar_128" t-att-title="activityView.activity.assignee.nameOrDisplayName" t-att-alt="activityView.activity.assignee.nameOrDisplayName"/>
```

Comment:
Its missing `me-2` that was in the CSS file, the avatar is too close to the name.